src/account/azext_account/generated/action.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import argparse
+from knack.util import CLIError
+
+
+# pylint: disable=protected-access
+
+
+class AddOwners(argparse._AppendAction):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        super(AddOwners, self).__call__(parser, namespace, action, option_string)
+
+    def get_action(self, values, option_string):  # pylint: disable=no-self-use
+        try:
+            properties = dict(x.split('=', 1) for x in values)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+            if kl == 'object_id':
+                d['object_id'] = v
```

Comment:
If the action only has one property `object_id`, we may pass it directly and can get rid of the action.

Reply:
yeah, that should be deleted, not used